front/src/reducers/osrdconf/osrdConfCommon/__tests__/utils.ts

```diff
@@ -358,16 +358,75 @@ const testCommonConfReducers = (slice: OperationalStudiesConfSlice | StdcmConfSl
     expect(state.vias).toBe(newVias);
   });
 
-  it('should handle addVias', () => {
-    const via1 = testDataBuilder.buildPointOnMap({ id: 'via-1' });
-    const via2 = testDataBuilder.buildPointOnMap({ id: 'via-2' });
-    const store = createStore(slice, {
-      vias: [via1],
+  describe('should handle addVias', () => {
```

Comment:
Is it worth to test both the action and the function as they do the same thing ? Maybe the action is enough ?

Reply:
Testing both helps us catch more mistakes. The action test checks integration with Redux and state updates, while the function test focuses on specific business logic. This way, we make our code stronger and find problems easier.

Reply:
Ok fine by me :)